src/RobotInterface/YarpImplementation/src/YarpRobotControl.cpp

```diff
@@ -451,20 +457,26 @@ struct YarpRobotControl::Impl
 
             } else if (mode == IRobotControl::ControlMode::Position)
             {
-                std::vector<double> refSpeeds(indeces.size());
                 for (int i = 0; i < indeces.size(); i++)
                 {
                     const auto jointError = std::abs(jointValues[indeces[i]]
                                                      - this->positionFeedback[indeces[i]]);
 
                     constexpr double scaling = 180 / M_PI;
                     constexpr double maxVelocityInDegPerSeconds = 3.0;
-                    refSpeeds[i] = std::max(maxVelocityInDegPerSeconds,
-                                            scaling * (jointError / this->positioningDuration));
+                    this->positionControlRefSpeeds[i]
+                        = std::max(maxVelocityInDegPerSeconds,
+                                   scaling * (jointError / this->positioningDuration));
```

Comment:
I am not sure of this logic, but this is not part of what is changed by the PR.

Reply:
Actually, ``maxVelocityInDegPerSeconds`` is misleading, as it is instead the minimum velocity. Each joint is supposed to reach the desired position in the time specified by ``positioningDuration``. If the joint is already close to the desired configuration, we don't want the joint to go too slow, so we set the desired velocity to ``maxVelocityInDegPerSeconds``, that should be called ``minVelocityInDegPerSeconds``.

Reply:
Thank you @S-Dafarra I reneame `maxVelocityInDegPerSeconds` into `minVelocityInDegPerSeconds`